server/protocols/jmap/src/main/java/org/apache/james/jmap/utils/SystemMailboxesProviderImpl.java

```diff
@@ -41,7 +42,7 @@
     private final MailboxManager mailboxManager;
 
     @Inject
-    @VisibleForTesting SystemMailboxesProviderImpl(MailboxManager mailboxManager) {
+    public SystemMailboxesProviderImpl(MailboxManager mailboxManager) {
```

Comment:
As I thought, we add "VisibleForTesting" if we only want to use/test that method. In this case, we do not testing constructor function so I removed it.
Do I need to roll it back?